pkg/monitors/ntp/metadata.yaml

```diff
@@ -0,0 +1,17 @@
+monitors:
+- doc: |
+    This monitor will retrieve clock offset from ntp server.
+
+    It enforces a minimum interval of 15m.
```

Comment:
```suggestion
    It enforces a minimum interval of 30m.
```

Reply:
Ok, this is my last request and then I'm good with it.

Reply:
oh good catch! 
It is true I choose 15m first without any reason then I checked any best practice and see the pool ntp org one so I changed.